internal/service/mwaa/environment.go

```diff
@@ -15,12 +18,16 @@ import (
 	"github.com/hashicorp/terraform-provider-aws/internal/verify"
 )
 
+const (
+	validationRetryTimeout = 30 * time.Second
```

Comment:
For IAM consistency timeout we have been naming this `propagationTimeout = 2 * time.Minute`.

Reply:
Updated.